examples/citrix_adc/README.md

```diff
@@ -0,0 +1,369 @@
+-->
+
+## Who should use this example? <!-- Suggest not using "Persona" as this is an UX term used more internally in software dev than a term that users would typically understand; while more techy people will use your example and might understand UX notion of personas, they are in this context a user. -->
+
+The steps described in this example are typically performed by **DevOps engineers** or **system administrators**. Generally, you'll need a basic understanding of Citrix ADC, Venafi Trust Protection Platform or Venafi Cloud, and the required permissions for completing the tasks described in the example.
+
+<!--
+DW: So I suggest adding--as I tried to do in that second sentence--the basic knowledge (as well as the permissions and access to the various systems) that is required in order to successfully complete your example. 
+-->
+## About this example <!--To make this more conversational and friendly, I've changed the title from "Solution" to this one. -->
+
+<!-- ORIGINAL TEXT: In order to increase reliability and capacity of applications, an application delivery controller(ADC) manages web traffic of your server application into nodes in order to reduce the "weight load" of those applications.
+-->
+<!-- DW: I took this first para out because I don't think we need to describe in this section what ADCs are and what they do. -->
+
+In this example, we use Terraform's _infrastructure as code_ automation process with the _Venafi Provider_ to generate and install certificates as part of SSL termination on a load balancer (Citrix ADC). We'll also utilize three HTTP servers contained in a cluster as the endpoints that are sending and receiving web traffic and being managed by Citrix ADC.
+
+<!-- **DW:** The original paragraph above wasn't clear to me; in my attempt to undersand it, I've written a new para. If I've lost the technical meaning, it's because I couldn't follow the original logic. Some of the questions I had from the original were these: Which parts of the explanation are Terraform's and which parts are Venafi...because the first half of the original sentence made it sound like Terraform has an automated process already for generating and installing certs, and so why woud you need Venafi? But I knew that's not true. So I wondered if it was saying that the Venafi Provider, as a service component of Terraform, is creating/installing the certs? In short, I wasn't clear which parts are us and which parts are Terraform, etc. And understanding that will I think help users stay oriented to "who's doing what" as they prepare to test drive your example. -->
```

Comment:
We should change the "load balancer" to "application delevery controller"  or "ADC" since this is modern term for load balancer. Perhaps we could rephrase it from:

"to generate and install certificates as part of SSL termination on a load balancer (Citrix ADC)"

to:

"to generate and install certificates as part of SSL termination on a Application Delevery Controller (Citrix ADC) for load balancing web traffic."

Reply:
I agree, Luis...with just a couple of minor edits:

< . . . to generate and install certificates as part of SSL termination on an ADC (specifically, Citrix ADC) for load balancing web traffic.

Because we already defined application delivery controller as ADC earlier in the document, I used ADC here, per our style guide.